app/models/bucket-list.js

```diff
@@ -37,6 +37,14 @@ var BucketList = CachedList.extend({
     cluster: DS.belongsTo('cluster'),
 
     /**
+     * The max amount of records that should be fetched in a given request
+     * @property maxBucketsPerRequest
+     * @type Integer
+     * @default 100
+     */
+    maxBucketsPerRequest: DS.attr('integer', { defaultValue: 100 }),
```

Comment:
This should be put on inherited cached list model. Will update with rest of feedback.